Content.Shared/Pointing/PointingEvents.cs

```diff
@@ -31,3 +32,12 @@ public PointingAttemptEvent(NetEntity target)
 /// <param name="Pointer"></param>
 [ByRefEvent]
 public readonly record struct AfterGotPointedAtEvent(EntityUid Pointer);
+
+// Begin DeltaV Additions - Interactions with tile pointing
+/// <summary>
+/// Raised on the entity who is pointing after they point at at a tile.
+/// </summary>
+/// <param name="Pointed">MapCoordinates pointed at by the entity.</param>
+[ByRefEvent]
+public readonly record struct AfterPointedAtTileEvent(MapCoordinates Pointed);
+// End DeltaV Additions - Interactions with tile pointing
```

Comment:
Summary comments are a special case you can consolidate

```suggestion
/// <summary>
/// DeltaV - Raised on the entity who is pointing after they point at at a tile.
/// </summary>
/// <param name="Pointed">MapCoordinates pointed at by the entity.</param>
[ByRefEvent]
public readonly record struct AfterPointedAtTileEvent(MapCoordinates Pointed);
```

Reply:
Oh? Even if I've added the record struct to the file?

Reply:
Yeah, one-line declarations -- this, or new variables on upstream components -- you can just put DeltaV in the summary comment.

Although now that I think of it, our upstream mergers probably would appreciate this one just being moved to a _DV/Pointing/PointingEvents.cs file

Reply:
Yeah, that's fair and easier to do. Cheers.